app/src/features/admin/users/ActiveUsersList.tsx

```diff
@@ -170,20 +181,60 @@ const ActiveUsersList: React.FC<IActiveUsersListProps> = (props) => {
         open: true
       });
 
-      props.getUsers(true);
+      props.refresh();
     } catch (error) {
       const apiError = error as APIError;
       showErrorDialog({ dialogText: apiError.message, dialogErrorDetails: apiError.errors, open: true });
     }
   };
 
+  const handleAddSystemUsersSave = async (values: IAddSystemUsersForm) => {
+    setOpenAddUserDialog(false);
+
+    try {
+      for (const participant of values.participants) {
+        await biohubApi.admin.addSystemUser(
+          participant.userIdentifier,
+          participant.identitySource,
+          participant.system_role
+        );
+      }
+
+      props.refresh();
+    } catch (error) {
+      dialogContext.setErrorDialog({ ...defaultErrorDialogProps, open: true, dialogErrorDetails: error });
```

Comment:
Getting a small warning about `error` being unknown.  Need to cast `error` to `APIError`. 

In general, (regarding `dialogError` and `dialogErrorDetails`) they aren't set consistently everywhere, but ideally they should both be set as below:

```typescript
      openErrorDialog({
        ...defaultErrorDialogProps,
        open: true,
        dialogError: (error as APIError).message,
        dialogErrorDetails: (error as APIError).errors
      });
```

Reply:
There are some tech debt improvements on the to-do list to audit the error handling in general.  And ensure the API returns sensible error message, etc.